snapshots.js

```diff
@@ -0,0 +1,8865 @@
+        "",
+        "",
+        "",
+        "",
+        "",
+        "",
+        "",
+        "",
+        "",
+        "",
+        "",
+        "N",
+        "",
+        "D",
+        "D",
+        "N",
+        "",
+        "DN",
+        "",
+        "D",
+        "D",
+        "",
+        "D",
+        "DN",
+        "N",
+        "",
+        "DN",
+        "",
+        "U",
+        "",
+        "",
+        "",
+        "",
+        "",
+        "",
+        "",
+        "",
+        "",
+        "",
+        "",
+        "",
+        "",
+        "",
+        "",
+        "",
+        "",
+        "",
+        "",
+        "",
+        "",
+        "",
+        "",
+        "",
+        "L4",
+        "",
+        "",
+        "",
+        "",
+        "",
+        "",
+        "",
+        "",
+        "",
+        "",
+        "",
+        "",
+        "",
+        "",
+        "",
+        "",
+        "",
+        "",
+        "",
+        "",
+        "",
+        "",
+        "",
+        "",
+        "",
+        "",
+        "",
+        "",
+        "",
+        "",
+        "",
+        "",
+        "",
+        "",
+        "",
+        "",
+        "",
+        "",
+        "",
+        "",
+        "",
+        "N",
+        "",
+        "D",
+        "N",
+        "N",
+        "",
+        "D",
+        "N",
+        "",
+        "D",
+        "N",
+        "P",
+        "",
+        "",
+        "",
+        "",
+        "D",
+        "DN",
+        "",
+        "D",
+        "N",
+        "",
+        "D",
+        "D",
+        "",
+        "U",
+        "",
+        "",
+        "",
+        "",
+        "",
+        "",
+        "",
+        "",
+        "",
+        "",
+        "0",
+        "0",
+        "0",
+        "0",
+        "0",
+        "0",
+        "24",
+        "24",
+        "24",
+        "24",
+        "24",
+        "24",
+        "24",
+        "24",
+        "24",
+        "24",
+        "24",
+        "24",
+        "24",
+        "24",
+        "24",
+        "24",
+        "24",
+        "24",
+        "24",
+        "24",
+        "24",
+        "24",
+        "24",
+        "24",
+        "24",
+        "24",
+        "24",
+        "24",
+        "24",
+        "24",
+        "0",
+        "0",
+        "0",
+        "0",
+        "0",
+        "0",
+        "0",
+        "0",
+        "0",
+        "0",
+        "0",
+        "0",
+        "10",
+        "10",
+        "10",
+        "10",
+        "10",
+        "10",
+        "10",
+        "10",
+        "10",
+        "10",
+        "10",
+        "10",
+        "10",
+        "10",
+        "10",
+        "10",
+        "10",
+        "10",
+        "10",
+        "10",
+        "10",
+        "10",
+        "10",
+        "10",
+        "10",
+        "10",
+        "10",
+        "10",
+        "10",
+        "10",
+        "0",
+        "0",
+        "0",
+        "0",
+        "0",
+        "0"
+      ],
+      "2": [
```

Comment:
Czy to jest stary snapshot.js?

Bo z tego co rozumiem, taki wpis do słownika jest dodawany w przypadku jeśli w jednym `it` my robimy więcej niż dwa testy. 
Ale nie widzę tego żeby tak rzeczywiście było. Chyba że coś źle rozumiem?

![image](https://user-images.githubusercontent.com/21079319/112902442-6cf7f780-90e6-11eb-8f5b-fcf305992889.png)

Reply:
Tak, rzeczywiście stary snapshot się zaplątał